src/config/dynamic-theme-fixes.config

```diff
@@ -3416,6 +3416,16 @@ INVERT
 
 ================================
 
+cad.onshape.com
+
+IGNORE INLINE STYLE
+#canvas
+
+CSS
+canvas { filter: invert(1) hue-rotate(180deg) !important; background: #fff !important; }
```

Comment:
```suggestion
INVERT
canvas

IGNORE INLINE STYLE
#canvas
```

Reply:
The suggested change didn't work as the background color needs to be changed. But I used the recommended changes to simplify it.